ansible/roles/neutron/templates/neutron.conf.j2

```diff
@@ -198,3 +198,8 @@ admin_password = {{ infoblox_admin_password }}
 wapi_version = {{ infoblox_wapi_version }}
 wapi_max_results = {{ infoblox_wapi_max_results }}
 {% endif %}
+
+{% if enable_ironic | bool %}
+[ironic]
+os_region = {{ openstack_region_name }}
```

Comment:
My understanding is that the ironic_neutron_agent.ini is automatically loaded into the server/agent if the Ironic neutron agent is configured. I think this means that this section can technically also go in the ironic_neutron_agent config. It's a bit weird that both `os_region` and `region_name` are needed, but I've seen this behavior elsewhere, in particular around Ironic I think. Does everything work if this moves to the ironic_neutron_agent.ini?

You should be able to test by just editing the local checkout of kolla-ansible on an existing deployment node.

Reply:
@diurnalist thank you! now I know what's going on here :) The documentation of ironic_neutron_agent has "region_name" as the region parameter name, but it's actually "os_region". I fixed it and tested it in uc@dev.